src/magnetVacuumFF.cpp

```diff
@@ -1,14 +1,7 @@
 // SPDX-License-Identifier: LGPL-3.0-or-later
-// Copyright (C) 2022 Alex Jentsch, Whitney Armstrong, Wouter Deconinck
-
-//==========================================================================
-//
-//      <detector name ="DetName" type="Beampipe" >
-//      <layer id="#(int)" inner_r="#(double)" outer_z="#(double)" >
-//      <slice material="string" thickness="#(double)" >
-//      </layer>
-//      </detector>
-//==========================================================================
+// Copyright (C) 2023 Alex Jentsch
```

Comment:
Same comment as above.

Reply:
Same as above - this has basically been completely re-written from scratch.